dhis-2/dhis-api/src/main/java/org/hisp/dhis/mapping/MapView.java

```diff
@@ -187,6 +187,8 @@
 
     private String eventCoordinateField;
 
+    private String organisationUnitCoordinateField;
```

Comment:
Actually we have a field already @vietnguyen , if you look at `BaseAnalyticalObject` we have a field `orgUnitField` already. So you can just map that `orgUnitField` field in the hibernate mapping but not add this field to `MapView`.

Reply:
fixed